### PR TITLE
feat(teams): add jacketNr to team model

### LINKED ISSUE #101: Separate out jacket number for running team and teamid
Currently, this is the same, which makes it hard to change jacket numbers of a team

### PR DESCRIPTION
**Pull request by NuttyShrimp**
closes #101 
goes hand in hand with https://github.com/12urenloop/manual-count-2/pull/62

### CODE REVIEW

File: src/main/java/telraam/database/models/Team.java [hunk 1]
```diff
@@ -9,6 +9,7 @@ public class Team {
     private Integer id;
     private String name;
     private Integer batonId;
+    private Integer jacketNr;
```
Comment (redfast00) [inline src/main/java/telraam/database/models/Team.java]: Hm, I think this should be a string, instead of an integer

**Comment by NuttyShrimp** [comment]
Also fixed the tests, made the default value in the model the same as in the DB, the constructors are only used in tests & unused lappers

File: src/main/java/telraam/database/models/Team.java [hunk 2]
```diff
@@ -9,7 +9,7 @@ public class Team {
     private Integer id;
     private String name;
     private Integer batonId;
-    private Integer jacketNr;
+    private String jacketNr = "0";
```
Comment (redfast00) [inline src/main/java/telraam/database/models/Team.java]: Als dat toch een default nodig heeft, dan mss '' (lege string) of 'INVALID' nemen?

Comment (FKD13) [reply src/main/java/telraam/database/models/Team.java]:
```suggestion
    private String jacketNr = "8==D";
```

**Review by redfast00 — APPROVED** [review]

**Pull request merged**